runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -9915,12 +9925,10 @@ class INTERPRETER_CLASS
 	VMINLINE VM_BytecodeAction
 	throwDefaultConflictForMemberName(REGISTER_ARGS_LIST)
 	{
-		/* Load the conflicting method and error message from this special target */
-		buildGenericSpecialStackFrame(REGISTER_ARGS, 0);
```

Comment:
Leave this in but use the jit frame flags:

buildGenericSpecialStackFrame(REGISTER_ARGS, jitStackFrameFlags(REGISTER_ARGS, 0));